playbooks/provision.yaml

```diff
@@ -7,11 +7,11 @@
   vars:
     openshift_master_url: "{{ lookup('env','OCP_URL') }}"
     token: "{{ lookup('env','OCP_TOKEN') }}"
-    THOTH_BACKEND_NAMESPACE: "{{ lookup('env','USER') }}-thoth-dev"
-    THOTH_MIDDLETIER_NAMESPACE: "{{ lookup('env','USER') }}-thoth-dev"
-    THOTH_FRONTEND_NAMESPACE: "{{ lookup('env','USER') }}-thoth-dev"
-    THOTH_INFRA_NAMESPACE: "{{ lookup('env','USER') }}-thoth-dev"
-    THOTH_DEPLOYMENT_NAME: "{{ lookup('env','USER') }}-thoth-dev"
+    THOTH_BACKEND_NAMESPACE: "thoth-test-core"
+    THOTH_MIDDLETIER_NAMESPACE: "thoth-test-core"
+    THOTH_FRONTEND_NAMESPACE: "thoth-test-core"
+    THOTH_INFRA_NAMESPACE: "thoth-test-core"
+    THOTH_DEPLOYMENT_NAME: "thoth-test-core"
```

Comment:
I would like to keep env vars here not to accidentally do changes on existing deployment (and if the deployment exists, it is user's deployment).

Reply:
then we should use some general env var rather than an user specific dev environment?!